app/models/work_version.rb

```diff
@@ -123,7 +123,7 @@ def label(id)
             allow_nil: true,
             format: {
               with: /^(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)(?:-((?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\.(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\+([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?$/, # rubocop:disable Layout/LineLength
-              message: 'Version names must be in semantic version format, ex. 1.0.0',
+              message: I18n.t('format'),
```

Comment:
This translation didn't work either.  `I18n.t('activerecord.errors.models.work_version.attributes.version_name.format')` worked for me.

Reply:
Yep - I did this wrong. Good catch!